src/main/java/org/apache/mesos/scheduler/DefaultScheduler.java

```diff
@@ -81,63 +79,67 @@ void awaitTermination() throws InterruptedException {
     }
 
     Plan getPlan() {
```

Comment:
I think `getPlans` is better as long as one can distinguish between plans by name as requested by @nickbp